### PR TITLE
Refactor: Pagination component

### PR DESCRIPTION
**Pull request by smyrn96**
## Description

The pagination component is completely reconstructed.

## Changes

- No buttons to traverse to a specific page, traversal is accomplished by clicking next or previous buttons.
- Selection is used to decide the size of the rows.
- A text that indicates in which page the user is and which is the size of the rows.

Task: https://app.asana.com/0/1202704662972653/1204765142791769

### CODE REVIEW

File: src/components/Pagination/styles.ts [hunk 1]
```diff
@@ -1,26 +1,93 @@
...
+
+        button {
+          color: ${pagination.textColor};
+          font-style: italic;
+          padding-inline-start: 1rem;
+
+          &:active,
+          &:hover {
+            background-color: ${pagination.boxShadowColor};
+          }
```
Comment (maryxan) [inline src/components/Pagination/styles.ts:38]: You dont need this according to design

File: src/components/Pagination/styles.ts [hunk 2]
```diff
@@ -1,26 +1,93 @@
...
+            width: 5px;
+          }
+
+          li {
+            padding: 0.5rem 1rem;
+
+            .is-selected {
+              font-weight: bold;
+            }
+          }
```
Comment (maryxan) [inline src/components/Pagination/styles.ts:75]: check formatting

**Review by maryxan — CHANGES_REQUESTED** [review]

**Review by maryxan — COMMENTED** [review]
Also merge with master!

File: src/icons/right-arrow-icon.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg width="7" height="12" viewBox="0 0 7 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (smyrn96) [reply src/icons/right-arrow-icon.svg]: Fixed: [4cec6ea](https://github.com/epignosis/gnosis/pull/158/commits/4cec6eab34eef29c0e353d1ba8ca0e967ef39563)

File: src/components/Pagination/Pagination.tsx [hunk 4]
```diff
@@ -1,83 +1,119 @@
...
-        {paginationRange.map((pageNumber) => {
-          if (pageNumber === ellipsis) {
-            return ellipsis;
-          }
+          <div className="pagination-options">
+            <div className="dropdown" ref={wrapperRef}>
+              <div className="dropdown-button" onClick={toggleList}>
+                <Button iconAfter={RightArrowSVG} variant="outline">
+                  <Text fontSize="md">{selectionText}</Text>
+                </Button>
```
[the remark this answers is not in the document]
Comment (smyrn96) [reply src/components/Pagination/Pagination.tsx]: Fixed: [4cec6ea](https://github.com/epignosis/gnosis/pull/158/commits/4cec6eab34eef29c0e353d1ba8ca0e967ef39563)

File: src/components/Pagination/styles.ts (hunk 1 above)
Comment (smyrn96) [reply src/components/Pagination/styles.ts]: Fixed: [4cec6ea](https://github.com/epignosis/gnosis/pull/158/commits/4cec6eab34eef29c0e353d1ba8ca0e967ef39563)

File: src/components/Pagination/styles.ts (hunk 2 above)
Comment (smyrn96) [reply src/components/Pagination/styles.ts]: Fixed: [4cec6ea](https://github.com/epignosis/gnosis/pull/158/commits/4cec6eab34eef29c0e353d1ba8ca0e967ef39563)

**Review by maryxan — APPROVED** [review]

**Pull request merged**